torch/_torch_docs.py

```diff
@@ -2610,7 +2610,14 @@ def merge_dicts(*dicts):
 Args:
     input (Tensor): the dividend
     other (Tensor or Number): the divisor
+    rounding_mode (str, optional): Type of rounding applied to the result.
 
+    * ``"true"`` - no rounding and integers are promoted to floating point (default)
+                   Equivalent to the python 3 division operator: ``/``.
+    * ``"trunc"`` - truncate decimals from the true result, no type promotion.
```

Comment:
"truncate decimals from the true result, no type promotion" -> "rounds the results of the division towards zero."

?